graviti/portex/base.py

```diff
@@ -249,7 +249,7 @@ class PortexRecordBase(
 
     @property
     def _data(self) -> "ConnectedFields":  # type: ignore[override]
-        return self._fields_factory(**{name: getattr(self, name) for name in self.params})
+        return self._fields_factory({name: getattr(self, name) for name in self.params})
```

Comment:
shall we use `perf` rather than `refactor`?
also shall we add perf compare result in the commit message body?